### PR TITLE
feat: do not run konflux image-controller and build-service checks in PR

### PR DESCRIPTION
**Pull request by flacatus**

### CODE REVIEW

**Comment by flacatus** [comment]
@psturc could you take a look

File: ci-operator/config/konflux-ci/build-service/konflux-ci-build-service-main.yaml [hunk 1]
```diff
@@ -35,6 +35,7 @@ tests:
     product: ocp
     timeout: 1h0m0s
     version: "4.15"
+  optional: true
   skip_if_only_changed: ^docs/|^\.github/|^\.tekton/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
```
Comment (psturc) [inline ci-operator/config/konflux-ci/build-service/konflux-ci-build-service-main.yaml:39]: I think this needs to be removed as well

File: ci-operator/config/konflux-ci/image-controller/konflux-ci-image-controller-main.yaml [hunk 2]
```diff
@@ -35,6 +35,7 @@ tests:
     product: ocp
     timeout: 1h0m0s
     version: "4.15"
+  optional: true
   skip_if_only_changed: ^docs/|^\.github/|^\.tekton/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
```
Comment (psturc) [inline ci-operator/config/konflux-ci/image-controller/konflux-ci-image-controller-main.yaml:39]: same as above

**Review by psturc — CHANGES_REQUESTED** [review]

**Review by psturc — APPROVED** [review]
/lgtm

**Pull request closed without merging**